youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -143,7 +143,11 @@ def run_ffmpeg_multiple_files(self, input_paths, out_path, opts):
             stderr = stderr.decode('utf-8', 'replace')
             msg = stderr.strip().split('\n')[-1]
             raise FFmpegPostProcessorError(msg)
-        os.utime(encodeFilename(out_path), (oldest_mtime, oldest_mtime))
+        try:
+            os.utime(encodeFilename(out_path), (oldest_mtime, oldest_mtime))
+        except:
```

Comment:
it should be `except Exception` (or something more specific), otherwise `KeyboardInterrupt` will also be catched.